polytope/polytope.py

```diff
@@ -78,7 +78,10 @@
     logger.warn(
         '`polytope` failed to import `cvxopt.glpk`.\n'
         'Will use `scipy.optimize.linprog`.')
-
+try:
+    xrange
```

Comment:
At the time, it seemed like the most reasonable way to get python 3.x support. Otherwise, you have to go through and replace all the `xrange` with `range` or you have `from __past__ import xrange`. The former was more meddling than I was compfortable doing, and the later seemed silly because it requires a dependency.